### PR TITLE
flho-227 storage-weapons-ammo step

### PR DESCRIPTION
**Pull request by bendiggle**
* Added the step to the routes config and added forking to the supporting docs step to get to this step,
	* Added the fields to the fields config,
	* Added the translations for validation, pages and fields,
	* Added acceptance tests for the step and its page helpers

### CODE REVIEW

**Comment by bendiggle** [comment]
screen shot 2016-10-11 at 17 04 23

**Comment by easternbloc** [comment]
lets re-enable the CI before merging this stuffs

File: apps/new-dealer/acceptance/pages/storage-weapons-ammo.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+  url: 'storage-weapons-ammo',
+  'weapons-ammo-group': '#storage-weapons-ammo-group',
+  'weapons': '#storage-weapons-ammo-weapons',
```
Comment (joefitter) [inline apps/new-dealer/acceptance/pages/storage-weapons-ammo.js]: `'` 🚫

File: apps/new-dealer/acceptance/pages/storage-weapons-ammo.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  'ammunition': '#storage-weapons-ammo-ammunition'
```
Comment (joefitter) [inline apps/new-dealer/acceptance/pages/storage-weapons-ammo.js]: :no_good_man:

File: apps/new-dealer/index.js [hunk 3]
```diff
@@ -74,12 +76,40 @@ module.exports = {
...
+        target: '/storage-address',
+        condition(req) {
+          const stored = storedOnPremises(req, 'true');
+          const weaponsAndAmmo = weapons(req) && ammunition(req);
+          return stored && !weaponsAndAmmo;
+        }
+      }, {
+        target: '/weapons',
+        condition(req) {
+          const stored = storedOnPremises(req, 'false');
```
Comment (joefitter) [inline apps/new-dealer/index.js]: hey Friend, I think this is a confusing use case for this function as it acts differently in different cases. It's like you're telling it what to expect.

Could you refactor the storedOnPremises function to remove the second arg?
```js
function storedOnPremesis(req) {
  return req.sessionModel.get('stored-on-premesis') === 'true';
}
```

File: apps/new-dealer/index.js [hunk 4]
```diff
@@ -74,12 +76,40 @@ module.exports = {
...
+          return stored && weapons(req);
```
Comment (joefitter) [inline apps/new-dealer/index.js]: Then here you'd check for `!stored`

**Pull request merged**